crates/services/txpool/src/txpool/tests.rs

```diff
@@ -83,14 +95,15 @@ async fn insert_simple_tx_succeeds() {
 #[tokio::test]
 async fn insert_simple_tx_with_blacklisted_utxo_id_fails() {
     let mut context = TextContext::default();
+    let gas_price = 0;
 
     let (_, gas_coin) = context.setup_coin();
     let tx = TransactionBuilder::script(vec![], vec![])
         .script_gas_limit(GAS_LIMIT)
         .add_input(gas_coin.clone())
         .finalize_as_transaction();
     let mut txpool = context.build();
-    let tx = check_unwrap_tx(tx, &txpool.config).await;
+    let tx = check_unwrap_tx(tx, &txpool.config, gas_price).await;
```

Comment:
The `check_unwrap_tx` function is a test helper function. If you are using zero gas price everywhere, it is better to use zero inside of this function without modifying the signature.

In the tests where gas price matters, you can use another function=)